source/_integrations/blebox.markdown

```
@@ -528,6 +560,38 @@
+5. Copy the webhook URL to clipboard by clicking the "copy" icon next to the webhook ID
+6. and save it for later reference
+7. If applicable, add any desired conditions (the *And if* section) and actions
+   (the *Then do* section)
+
+Note: The webhook ID will be later needed in phase two and will have to be entered
+into the wBox mobile app. You may decide to use more convenient text value but remember
+that this is the only thing that authenticates webhooks within your network. Treat
+this ID like a password.
+
+Phase two requires configuring the device in wBox app:
+1. Configure the Blebox device (actionBox, actionBoxS, or proxiBox) by adding action of
```

Markdownlint

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)